### PR TITLE
chore(example): remove @cloudoperators/juno-utils dependency

### PR DESCRIPTION
**Pull request by taymoor89**
# Summary

This PR removes `@cloudoperators/juno-utils` dependency from `example` app and uses`msw` to intercept and mock network requests. This way we do not need to mock `fetch` API at all rather the service worker created by `msw` intercepts the network calls and executes our mock request handlers 😎 

# Changes Made

- Removes `@cloudoperators/juno-utils` dependencies
- Uses `msw` to mock networks responses.

# Related Issues

- #631 

# Screenshots (if applicable)

See there is a proper call to the endpoint in the network tab but it was intercepted by the mock service worker.

Screenshot 2024-12-05 at 14 30 18

# Testing Instructions

Test that example app is working as if there is no change.

# Checklist

- [x] I have performed a self-review of my code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have made corresponding changes to the documentation (if applicable).
- [x] My changes generate no new warnings or errors.

### CODE REVIEW

File: apps/example/public/mockServiceWorker.js [hunk 1]
```diff
@@ -0,0 +1,292 @@
...
+/* tslint:disable */
+
+/**
+ * Mock Service Worker.
+ * @see https://github.com/mswjs/msw
+ * - Please do NOT modify this file.
+ * - Please do NOT serve this file on production.
+ */
+
+const PACKAGE_VERSION = "2.6.6"
```
Comment (barsukov) [inline apps/example/public/mockServiceWorker.js:11]: Do we really need this whole logic?

Comment (taymoor89) [reply apps/example/public/mockServiceWorker.js:11]: I see your point but this file is actually auto-generated by `msw` as part of the setup and ideally should not to be changed.

https://mswjs.io/docs/integrations/browser

**Comment by ArtieReus** [comment]
Hi @taymoor89!! Cool thing! Thx!! Just wanted to mention before I review that I tried the preview and when creating a new Peak I get following error:

```text
Failed to load resource: the server responded with a status of 405 ()
```

Screenshot 2024-12-10 at 14 09 52

**Review by ArtieReus — APPROVED** [review]
Lgtm. Thx!!!!!!!!!!!!!

**Review by andypf — APPROVED** [review]

**Review by barsukov — APPROVED** [review]
LGTM

**Pull request merged**